satellite/satellitedb/utils.go

```diff
@@ -24,6 +25,26 @@ func bytesToUUID(data []byte) (uuid.UUID, error) {
 	return id, nil
 }
 
+// null time helps convert nil to time.Time
+type nullTime struct {
```

Comment:
This and its helper functions should go into our date helper package in internal

Reply:
`internal/dbutil` is it appropriate place to move this struct?